rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -54,7 +54,83 @@ paths:
           $ref: '#/components/responses/NotImplemented'
         default:
           $ref: '#/components/responses/DefaultError'
-
+  /transform/{TransformId}:
+    parameters:
+      - $ref: '#/components/parameters/TransformId'
+    get:
+      summary: Poll transformation request status
+      description: >
+        After posting a transformation request, the client can use the returned URI to poll for the transform's completion at this endpoint.
+      operationId: poll
+      tags:
+        - transform
+      responses:
+        '200':
+          $ref: '#/components/responses/TransformationResponse'
```

Comment:
✏️ I would expect TransformationResponse to be returned by the Submit transformation request endpoint, not the status endpoint. I think TransformPollResponse or TransformStatus is clearer.